core/src/components/Resizable.scss

```diff
@@ -17,8 +17,12 @@
   }
   &-border {
     position: absolute;
-    background-color: #999;
-    transition: .1s;
```

Comment:
Why remove the transition?

Reply:
Making the transformation instant can improve ux for smaller border width, making it easier to hold and drag.